homeassistant/components/rainmachine/services.yaml

```
@@ -97,6 +97,37 @@ unpause_watering:
       selector:
         device:
           integration: rainmachine
+push_flow_meter_data:
+  name: Push Flow Meter Data
+  description: Push Flow Meter data from Home Assistant to the RainMachine device.
```

Let's genericize this (since "pushing from Home Assistant" might be confusing).

```suggestion
  description: Push flow meter data to the RainMachine device.
```